spec/fixtures/stripe_webhooks/customer.subscription.updated-renewed.json

```diff
@@ -65,7 +65,8 @@
     },
     "previous_attributes": {
       "current_period_start": 1378488561,
-      "current_period_end": 1381076961
+      "current_period_end": 1381076961,
+      "latest_invoice": "inv_1234"
```

Comment:
Sorry to be picky, but a genuine invoice ID is prefixed `in_`, not `inv_`. 

```
…
"latest_invoice"=>"in_1DxCXwLoAAr9vREDACTED"
…
```

I think its worth using the same prefix to avoid any future confusion. Sorry I missed that first time around.